doc/content/examples/example5a.md

```diff
@@ -0,0 +1,121 @@
+# Example 5a: Dynamic Analysis of a Simplified Model of a Nuclear Power Plant Structure
+
+## Model Description
+
+This example demonstrates a dynamic analysis of a simplified stick model of a nuclear power plant taken from the SASSI2000 user manual [!citet](sassiuser2006)
+in MASTODON. The actual structure consists of a containment made up of prestressed concrete and a reinforced concrete internal structure. The simplified numerical model [fig:stickmodel] consists of two sets of stick models, one for the containment and the other for the internal structure. The lumped masses and the section properties of the stick model are presented in the right part of [fig:stickmodel].
+The superstructure is connected to the rigid foundation by four rigid foundation beams whose deformation is zero as shown in [fig:stickwithfoundation].
+MASTODON at present does not have the rigid elements, thus, the foundation beams and the solid foundation are made rigid by increasing the value of Young's Modulus of Elasticity by ${10}^3$. An acceleration time history is applied at the base of the foundation in the X, Y and Z direction and the response spectra at different elevations are compared with the response obtained from LS-DYNA model taken from [!citet](fragilityanalysis).
+
+
+!media media/examples/stickmodel.png
+       style=width:100%;margin-right:0px;float:center;
+       id=fig:stickmodel
+       caption=Illustration of the representative nuclear power plant structures and the corresponding stick models (left) and section properties and lumped masses of stick model (right) [!citep](sassiuser2006).
+
+!media media/examples/stickwithfoundation.png
+       style=width:50%;margin-left:150px;float:center;
+       id=fig:stickwithfoundation
+       caption=Illustration of the connection between the stick model and foundation [!citep](sassiuser2006).             
+
+!alert note
+- The horizontal separation between the stick-mass model in the above figures is only for illustration. Both the stick-mass models are located along the center line and the base node is shared by both the stick mass models.
+- The lumped mass value of mass no. 3 is 4600 kips and not 46000, and mass no. 2, which is omitted in the table is equal to 4200 kips.
+
+## Modeling in MASTODON
+
+!listing examples/ex05a/StickModel_accel_base.i
+
+The mesh is generated by importing an existing mesh file. The details of the types of supported mesh files can be found at [Getting Started](manuals/user/index.md). The stick-models and the foundation beams are modeled using 1D line elements and the foundation is modeled using 3D solid elements as shown in [fig:ex05input]. The displacements and rotations in X, Y and Z directions are defined in the variables section and the velocity, acceleration, rotational velocity and rotational acceleration are defined as the auxiliary variables.
+
+!media media/examples/ex05input.png
+       style=width:50%;margin-left:150px;float:center;
```

Comment:
This figure is not centered right now. Maybe because of the margin. Try to play around and center it.